kyuubi-common/src/main/scala/org/apache/kyuubi/config/KyuubiConf.scala

```diff
@@ -608,11 +608,21 @@ object KyuubiConf {
 
   val ENGINE_INITIALIZE_SQL: ConfigEntry[String] = buildConf("engine.initialize.sql")
     .doc("SemiColon-separated list of SQL statements to be initialized in the newly created " +
-      "engine before queries.")
+      "engine before queries. This configuration can not be used in JDBC url due to " +
+      "the limitation of Beeline/JDBC driver.")
     .version("1.2.0")
     .stringConf
     .createWithDefault("SHOW DATABASES")
 
+  val ENGINE_SESSION_INITIALIZE_SQL: ConfigEntry[String] =
+    buildConf("engine.session.initialize.sql")
+      .doc("SemiColon-separated list of SQL statements to be initialized in the newly created " +
+        "engine session before queries. This configuration can not be used in JDBC url due to " +
+        "the limitation of Beeline/JDBC driver.")
+      .version("1.3.0")
+      .stringConf
+      .createWithDefault("SHOW DATABASES")
```

Comment:
should we use `.fallbackConf(ENGINE_INITIALIZE_SQL)`?

Reply:
i think there are two things, fallback doesn't make sense to me.

Reply:
ok